fe/fe-core/src/test/java/com/starrocks/analysis/StorageVolumeTest.java

```diff
@@ -110,7 +110,7 @@ public void testShowStorageVolumesParserAndAnalyzer() {
 
     @Test
     public void testDescStorageVolumeParserAndAnalyzer() {
-        new MockUp<StorageVolumeMgr>() {
+        new MockUp<SharedNothingStorageVolumeMgr>() {
```

Comment:
shall be able to test both SharedNothingStorageVolumeMgr and SharedDataStorageVolumeMgr by setting different runMode.

better leave a TODO here.

Reply:
IMO, check one of them is enough.